ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,7 @@ Tell us what you think should happen.
 ### Actual behavior
 
 If possible, please create a small demo that demonstrates the issue.
+You can fork https://jsfiddle.net/marionettejs/adhv48ky/ for quick demo setup.  
```

Comment:
I didn't want to mention what versions of what the fork includes to make updating the fork easier and not have to continuously update this file for every upgrade.

I suggest a look at the external versions to make sure those are good... but like I said, easy to update.